WPThemeReview/Sniffs/Privacy/ShortenedURLsSniff.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * WPThemeReview Coding Standard.
+ *
+ * @package WPTRT\WPThemeReview
+ * @link    https://github.com/WPTRT/WPThemeReview
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WPThemeReview\Sniffs\Privacy;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * Detect the use of shortened URLs.
+ *
+ * Detection is based on a list of banned URL shortener services.
+ *
+ * @link https://make.wordpress.org/themes/handbook/review/required/#privacy
+ *
+ * @since 0.2.0
+ */
+class ShortenedURLsSniff implements Sniff {
+
+	/**
+	 * Error message template.
+	 *
+	 * @since 0.2.0
+	 *
+	 * @var string
+	 */
+	const ERROR_MSG = 'Shortened URLs are not allowed in the theme. Found: "%s".';
+
+	/**
+	 * Regex template.
+	 *
+	 * Will be parsed together with the url_shorteners blacklist in the register() method.
+	 *
+	 * @var string
+	 */
+	const REGEX_TEMPLATE = '`(?:%s)/[^\s\'"]+`i';
+
+	/**
+	 * Regex pattern.
+	 *
+	 * @since 0.2.0
+	 *
+	 * @var string
+	 */
+	private $regex = '';
+
+	/**
+	 * Found used shortener in a file.
+	 *
+	 * @since 0.2.0
+	 *
+	 * @var string
+	 */
+	protected $shortener;
+
+	/**
+	 * Supported Tokenizers.
```

Comment:
Not a hard rule, but it is customary to order properties `public`, `protected`, `private` (unless there is some reason why it really makes sense to mix it up).